src/Controllers/SCStormRunoffController.ts

```diff
@@ -95,25 +116,483 @@ module StreamStats.Controllers {
         public set SelectedAEP(val) {
             this._selectedAEP = val;
         }
-        private _selectedTab: SCStormRunoffType;
-        public get SelectedTab(): SCStormRunoffType {
-            return this._selectedTab;
+
+        // Synthetic UH
+        private _selectedAEPSynthetic;
+        private _selectedStandardCurve;
+        private _selectedCNModification;
+        private _selectedTimeOfConcentration;
+        private _selectedRainfallDistribution;
+        public AEPOptionsSynthetic = [{
```

Comment:
Let's go ahead and have the AEP be blank by default too. Based on our experiences with showing off the other tab with users, it seems that sometimes people don't notice the AEP drop-down since it is already selected.

Reply:
If we are switching them to all be blank, could you check that the drop downs in the Urban Bohman method are also blank?

Reply:
Ok great - I can remove the AEP default value from the Urban Bohman tab for the next push